src/sparseml/INTEGRATIONS.md

```diff
@@ -0,0 +1,56 @@
+# Training
```

Comment:
Should i name this file README.md so github auto displays it when you're in the sparseml directory?

Reply:
not right now, but maybe when we decide on a landing page